src/calculateRentalCost.js

```diff
@@ -4,7 +4,25 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const basePrise = 40;
+  const discontSevenDay = 50;
+  const discontThreeDay = 20;
+
+  if (days >= 7) {
+    return basePrise * days - discontSevenDay;
+  }
+
+  if (days === 6) {
+    return basePrise * days - discontThreeDay;
```

Comment:
The explicit check for `days === 6` is unnecessary because this case is already covered by the condition `days >= 3`. You can remove this `if` block to simplify the code.